graylog2-web-interface/src/routing/AppFacade.jsx

```diff
@@ -12,7 +12,7 @@ const CurrentUserStore = StoreProvider.getStore('CurrentUser');
 
 import 'bootstrap/dist/css/bootstrap.css';
 import 'font-awesome/css/font-awesome.css';
-import 'stylesheets/newfonts.less';
+import 'opensans-npm-webfont';
```

Comment:
Is this correct? The other imports all reference a specific CSS (or LESS) file while this only references the directory.